### PR TITLE
Update README.md title

### PR DESCRIPTION
**Pull request by ericksonl**

### CODE REVIEW

**Comment by peterj812** [comment]
I do not approve of the new name - Stonk was the choosen one!

**Comment by ericksonl** [comment]
Nuuu
We submitted it on DevPost under [Laughing Stock](https://devpost.com/software/quartus-primates-project)
And the submitted video (with the best audio quality) was titled [Laughing Stock - Quartus Primates - HackOR2021](https://www.youtube.com/watch?v=nLmeMCHBSA8)

**Comment by peterj812** [comment]
Let me rephrase I link the Stonk typo!

**Comment by ericksonl** [comment]
Brother, we let you keep the "room clearing farts" joke. Let me change the Readme to the correct project name.

**Review by platt-sam — APPROVED** [review]
👍 LGTM

**Comment by platt-sam** [comment]
**Suggestion:** We should remove the fart joke from the README and fix the other typos.

<sup>[https://conventionalcomments.org](https://conventionalcomments.org)</sup>

**Comment by peterj812** [comment]
Oh now we’re getting out of hand.

**Comment by ericksonl** [comment]
> **Suggestion:** We should remove the fart joke from the README and fix the other typos.
> 
> https://conventionalcomments.org

I could not agree more. Thank you so much Sam. This has been bothering me for years. 
Also, thank you for the link to conventionalcomments.org. This is an incredibly helpful tool.
I have updated the commit to include your suggestions, and fix other grammatical/spelling errors.
